c10/core/TensorOptions.h

```diff
@@ -691,6 +693,8 @@ inline DeviceType computeDeviceType(DispatchKey tid) {
     return DeviceType::XLA;
   } else if (tid == DispatchKey::MLC) {
     return DeviceType::MLC;
+  } else if (tid == DispatchKey::Meta) {
+    return DeviceType::Meta;
```

Comment:
I guess this change *technically* makes meta computations marginally slower, since now they have to go through BackendSelect to get to the meta kernel. Probably fine? Especially if we're hoping to improve BackendSelect perf soon anyway.

Reply:
Also I don't care about Meta perf ;)